### PR TITLE
fix: avoid corrupted export if data containing losted character \r

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: src/app/custom/exporters/README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+In this directory, it is possible to add configuration files (.ini) which will replace or complete the default files.
+To activate any modification it is necessary to stop and restart the docker container
+
+## Overload
+
+For example add a new file named csv.ini, it will replace the default file proposed by lodex.
+
+## Addition
+
+For example add a new file named xml.ini, activate this new exporter by adding its name to the relevant part of the configuration file (config.ini).
```
Comment (parmentf) [inline src/app/custom/exporters/README.md:12]: `config.ini` or `config.json`
I've never heard of `config.ini` in lodex!

File: src/app/custom/loaders/README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+In this directory, it is possible to add configuration files (.ini) which will replace or complete the default files.
+To activate any modification it is necessary to stop and restart the docker container
+
+## Overload
+
+For example add a new file named csv.ini, it will replace the default file proposed by lodex.
+
+## Addition
+
+For example add a new file named tbx.ini, activate this new loader by adding its name to the relevant part of the configuration file (config.ini).
```
Comment (parmentf) [inline src/app/custom/loaders/README.md:12]: `config.ini` or `config.json` ?

File: src/app/custom/routines/README.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+In this directory, it is possible to add configuration files (.ini) which will replace or complete the default files.
+To activate any modification it is necessary to stop and restart the docker container
+
+## Overload
+
+For example add a new file named graph-by.ini, it will replace the default file proposed by lodex.
+
+## Addition
+
+For example add a new file named network-by.ini, activate this new routine by adding its name to the relevant part of the configuration file (config.ini).
```
Comment (parmentf) [inline src/app/custom/routines/README.md:12]: `config.ini` or `config.json` ?

**Comment by touv** [comment]
config.json

**Pull request merged**